web_refresher/__manifest__.py

```diff
@@ -1,11 +1,11 @@
 {
     "name": "Web Refresher",
-    "version": "15.0.1.0.0",
-    "author": "Compassion Switzerland, Odoo Community Association (OCA)",
+    "version": "16.0.1.0.0",
+    "author": "Compassion Switzerland, FactorLibre, Odoo Community Association (OCA)",
```

Comment:
This migration doesn't qualify for being co-authors of the module. You should add to contributors instead.

Reply:
ok! Thanks for the review!, changes are already done